cookbooks/swift/templates/default/etc/swift/proxy-server/proxy-server.conf.d/20_settings.conf.erb

```diff
@@ -1,6 +1,9 @@
 [DEFAULT]
 bind_port = 8080
 
+[app:proxy-server]
+require_proxy_protocol = <%= @ssl %>
```

Comment:
I'm less and less certain that this is a good idea -- at least, it doesn't seem to make a lot of sense for a dev environment. Can't run probe tests like this! Though I like the idea of using PROXY protocol when terminating TLS... I don't know that I can swing it for this :disappointed:

Reply:
Got an upstream patch to resolve it -- I'm not so worried now.